### DISCUSSION TITLE
Set a new account expiry while resting the password using webclient UI

### DISCUSSION THREAD

**Discussion by rashmimaharana** [drakkan/sftpgo#708] [Ideas]
Hi @drakkan 
Thanks for your all feedback and response :)

Will be it good to implement below scenario in SFTPGo. I am not sure its already there. Will it be good to have ?
When i tried to reset the password, i observed its not updating the new expiry date. Manually we need to update the same from Admin UI.

1> Created sftp account named "test-sftpgo" with account expiry date ="10-Feb-2022"
2> User will login to client webUI and reset the password on "09-Feb-2022" before expiry.
3> The above step will update the new password and new account expiry date by adding default 90 calendar date (should be configurable) and new expiry date will be ="03-May-2022".
4> If account expiry date is blank , then no need to update new account expiry date. Leave it as blank.

**Comment by drakkan** [comment]
Hi,

I don't like password expiration policies, they are considered obsolete, for example read the "Before you begin" section [here](https://docs.microsoft.com/en-us/microsoft-365/admin/manage/set-password-expiration-policy?view=o365-worldwide). SFTPGo supports 2FA

**Reply by rashmimaharana** [reply]
Thanks @drakkan for your valuable response :)
